internal/pkg/cache/redis/redis.go

```diff
@@ -65,11 +95,11 @@ func NewRedisWithConfig(cfg *config.RedisConfig) *Redis {
 	}
 }
 
-// Parse the input Redis address by splitting the list of addresses separated by commas (,)
-func parseRedisAddress(address string) []string {
+// parseRedisClusterMultiAddress Parse the input Redis address by splitting the list of addresses separated by commas (,)
```

Comment:
Can we change this back to the original function name if this also works for both redis server and redis cluster?

Reply:
Sure thing